### ISSUE TITLE
Media picker field editor not displaying for a part

### ISSUE THREAD

**Issue by petedavis** [OrchardCMS/Orchard#6277]
Using Orchard CMS 1.9.2 (I havent tested on previous versions of Orchard)
Using the Featured Item Slider from this post: 
* http://pepfry.com/tutorials/image-slider-in-orchard-cms-featured-item-slider
* https://github.com/sunkist/FeaturedItemSlider

When Creating/Editing the Featured Item types, the editor for the media library picker field is not displayed.
![image](https://cloud.githubusercontent.com/assets/164331/12283177/3bc87d02-b9f4-11e5-82ab-620bed7e26d9.png)

However the field is displayed in the featured item list view in the admin area.
![image](https://cloud.githubusercontent.com/assets/164331/12283159/17f58e38-b9f4-11e5-9ffc-7f1cd21898c7.png)

If i edit the content type definition and add a media library picker field to the Featured Item content type, then the editor for this field is displayed, but obviously the module is looking for the field on the FeaturedItemPart of the FeaturedItem content type for which there is no editor.

Is this a regression in Orchard CMS since approx 1.7?

**Comment by sebastienros** [comment]
Is the same issue happening when you are not using the Featured Item module? For instance by creating a custom Part and adding the field to it?

**Comment by petedavis** [comment]
Yes it is. I just downloaded the code from the Advanced Orchard Pluralsight course, added a media picker field to the MoviePart from that project, and same result. 

The reason I used this code is that the MoviePart has a Taxonomy field attached, and the MoviePart is added to the Movie Content Type, so same structure, and editing/creating shows the editor for the taxonomy field. However adding a media picker field does not show when editing a movie.

The only change from the code was the following migration
```
        public int UpdateFrom8() {
            ContentDefinitionManager.AlterPartDefinition("MoviePart",
                cfg => cfg
                    .WithField("Picture", fld => fld
                    .OfType("MediaPickerField")
                    .WithSetting("MediaLibraryPickerFieldSettings.Multiple", "False")));
            return 9;
        }
```

**Comment by LivyChops** [comment]
Ha, I'm experiencing the same thing with basically the same module [I extended the FeatureItemSlider a while ago to include a layout so the user could create whatever content he felt like] and all worked as expected with 1.9 to (1-x for a while)? Though since 1.9.2, it behaves as @petedavis said. I haven't had time to look at it properly, but note straightaway that the elements are actually being rendered, though an in-line style of 'display: none' is applied to the media-library-picker-field.  So clearly CSS needs to be investigated and I'm thinking something with Layouts?  At first I thought it was a placement issue, but knowing that it used to work and due to 1.9.2 [layouts addition] and the fact I can get the mediapicker to display with the featureelement by themselves but only via:

    <Place Parts_FeaturedItem_Edit="Content:after"/> -->

and not my original code which was simply:

    <Place Fields_MediaLibraryPicker_Edit="Content:1"/>
    <Place Parts_FeaturedItem_Edit="Content:2"/>

where the layout editor was also visible [but I didn't need to declare in my placement file], now I can only ever get two out of the three to render but never all at the same time like once possible [even if I add Parts_Layout_Edit to the placement]?

**Comment by kcastro18** [comment]
Does anybody know how to solve this yet? I'm having the same problem. I'm using Orchard 1.10.2.
